alpha/apps/kaltura/lib/batch2/kFlowManager.php

```diff
@@ -686,6 +686,7 @@ public function objectChanged(BaseObject $object, array $modifiedColumns)
 			&& $object->getStatus() == entryStatus::READY)
```

Comment:
if condition never applies: 
check if we have cleanup for both batchJob and batchJobLock tables for the delayed jobs